enterprise/api/renderers.py

```diff
@@ -8,13 +8,18 @@
 from rest_framework.settings import api_settings
 
 
-class IgnoreClientContentNegotiation(DefaultContentNegotiation):
+class CustomContentNegotiation(DefaultContentNegotiation):
```

Comment:
Something that tries to describe what this custom content negotiation class does would be better.